Packs/PaloAltoNetworks_IoT3rdParty/Integrations/PaloAltoNetworksIoT3rdParty/PaloAltoNetworksIoT3rdParty.yml

```diff
@@ -0,0 +1,157 @@
+category: Utilities
+commonfields:
+  id: Palo Alto Networks IoT 3rd Party
+  version: -1
+configuration:
+- display: URL.
+  name: url
+  required: true
+  type: 0
+- display: Customer ID.
+  name: Customer ID
+  required: true
+  type: 0
+- display: Access Key.
+  name: Access Key
+  required: true
+  type: 0
+- display: Key ID.
+  name: Key ID
+  required: true
+  type: 0
```

Comment:
type: 0 is string. do not we want to encrypt it in the demisto's DB?
if so use type: 4